src/client/Game.jsx

```diff
@@ -157,6 +185,21 @@ export default class Game extends React.Component {
     }))
   }
 
+  rematch (action) {
```

Comment:
Change from noun to verb, for instance, `startRematch(...)`

Reply:
Was only relevant with the first commit, before request/accept was introduced. But perhaps "action" could be changed to "status" with requested/accepted to make it more clear.